testing/test_hash.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python
+
+"""
+Created by: Metannotate Team (2017)
+
+Description: A simple unittest for testing the hash.py module
+
+"""
+
+# Imports:
+from modules import hash
+import unittest
+
+
+class HashTesting(unittest.TestCase):
+    """
+    Description: A unit testing class for testing the hash.py module. Tests the functions 'hexhash' and 'md5hash'.
+    Called by nosetests in circleci's config.yml.
+    """
+
+    def test_hash(self):
+        """
+        Description: Method that tests the methods 'hexhash()' and 'md5hash()' from module hash
+
+        :param self: Reference to the HashTesting class
```

Comment:
@KyleBehiels We generally skip the ```self``` param as it is very common throughout the code. self is equivalent to ```this``` in other languages. Arbitrary rule I know.